docs/source/instruction_set_extensions.rst

```diff
@@ -20,17 +20,22 @@
 CORE-V Instruction Set Extensions
 =================================
 
-CV32E40P supports the following CORE-V ISA Extensions, which are part of **Xcorev** and can be enabled by setting ``PULP_XPULP`` == 1.
+CV32E40P supports the following CORE-V ISA Extensions, which can be enabled by setting ``PULP_XPULP`` == 1.
 
- * Post-Incrementing load and stores, see :ref:`corev_load_store`.
- * Hardware Loop extension, see :ref:`corev_hardware_loop`.
- * ALU extensions, see :ref:`corev_alu`.
- * Multiply-Accumulate extensions, see :ref:`corev_multiply_accumulate`.
- * Single Instruction Multiple Data (aka SIMD) extensions, see :ref:`corev_simd`.
+ * Post-Incrementing load and stores, see :ref:`corev_load_store`, invoked in the tool chain with ``-march=xcvmem``.
```

Comment:
Do we need to be more precise like ``-march=rv32i*_xcvmem``

Reply:
That's a good suggestion. I'll make this change.